README.md

```diff
@@ -18,9 +20,46 @@ If you'd prefer to use a local copy of the portage tree, you can point
 Licensing information can be found in the respective files, so consult
 them directly. Most ebuilds are licensed under the GPL version 2.
 
+Upstream Gentoo sources: http://sources.gentoo.org/gentoo-x86/
+
+# Removing packages
+
+Be careful when removing packages. The following section offers tips for preventing
+breakage, but they are by no means exhaustive. Gentoo's portage utilities wont take
+into account some pretty core packages like catalyst or glibc, so take extra care
```

Comment:
Is `package` the right way to describe the catalyst infrastructure?

Reply:
Probably not. It's not clear to me how exactly it's used. We have a catalyst package (`dev-util/catalyst`) but I can't find a single place we install it or use it.

Reply:
`scripts/build_library/catalyst.sh`

Reply:
After talking with @dm0-, jenkins installs this. I'm adding it to `sdk-extras` (https://github.com/coreos/coreos-overlay/pull/2867) so we can track it and its dependencies.